### PR TITLE
Clean up the run-all script.

### PR DESCRIPTION
**Pull request by awf**
Comment parameters.  Now ``get-help ADBench/run-all.ps1 -detailed`` gives useful descriptions.
Add "-repeat" option to rerun already-completed tests.
Add "-repeat_failures" to rerun only tests which timed out
Default sampling time to 10 seconds.

### CODE REVIEW

File: ADBench/run-all.ps1 [hunk 1]
```diff
@@ -181,9 +209,17 @@ Class Tool {
 		elseif ($objective.endswith("SPLIT")) { $out_name = "$($this.name)_split" }
 		else { $out_name = $this.name }
 		$output_file = "${dir_out}${fn}_times_${out_name}.txt"
-		if (!$script:repeat -and (Test-Path $output_file)) {
-			Write-Host "          Skipped test (already completed)"
-			return
+		if (Test-Path $output_file) {
```
Comment (tomjaguarpaw) [inline ADBench/run-all.ps1]: Wouldn't it be clearer to leave `!$script:repeat -and` in the outer `if`?  Nothing is run in the body unless `!$script:repeat`.  The body would become

```powershell
if ($script:repeat_failures) { ... }
else { ... }
```

**Comment by tomjaguarpaw** [comment]
Nice improvements!

File: ADBench/run-all.ps1 (hunk 1 above)
Comment (tomjaguarpaw) [reply ADBench/run-all.ps1]: I can go ahead and do that for you if you like.

Comment (awf) [reply ADBench/run-all.ps1]: Good change, please do!

**Comment by tomjaguarpaw** [comment]
(If you're happy I think we want to squash merge)

**Pull request merged**